src/reference/informative/informA1.rst

```diff
@@ -16,35 +16,39 @@
     In CellML, two namespaces are used.
     These are the CellML namespace itself (which helps to define the :code:`units` elements as distinct from the XML default units system) and the MathML namespace used to interpret the :code:`math` elements.
     For more information on namespaces in general, please refer to the :namespace_help:`W3 schools namespace page <>`, or for the MathML namespace please refer to the :mathml2help:`W3 MathML namespace <#inferf.namespace>` page.
-    
+
     .. container:: heading3
 
-      Unicode, Basic Latin, and European numerals
+      Unicode, Basic Latin alphabetical characters, and digits
 
     The Unicode project is an attempt to codify all of the symbols - alphabets, writings, even emojis - of all the languages of the world so that they can be interchangeable and interpreted by computers.
     Since computers understand numbers rather than symbols, each character in each language is given a unique numerical code.
-    The codes themselves are arranged into blocks representing sets or alphabets of characters, and the Basic Latin alphabet is one of these.
-    It contains the upper and lowercase alphabet without decoration:
+    The codes themselves are arranged into blocks representing sets of characters, and the Basic Latin block is one of these.
+    It contains, amongst other things, the upper and lowercase alphabet without decoration:
 
       :code:`abcdefghijklmnopqrstuvwxyz` (symbols between :unicode:`0061` and :unicode:`007A`)
       :code:`ABCDEFGHIJKLMNOPQRSTUVWXYZ` (symbols between :unicode:`0041` and :unicode:`005A`)
 
-    The European numerals are the Unicode set:
+    These characters are refferd to in CellML as the Basic Latin alphabetic characters.
+
+    The digits are the Unicode characters:
 
       :code:`0123456789` (symbols between :unicode:`0030` and :unicode:`0039`)
 
+    Together with the Basic Latin alphabetic characters, they form the Basic Latin alphanumerical characters.
```

Comment:
Is it "alphabetic" and "alphanumeric"  ... or "alphabetical" and "alphanumerical"? There doesn't seem to be any difference in the meaning of each of the pairs.